guides/release/components/template-lifecycle-dom-and-modifiers.md

```diff
@@ -243,45 +243,51 @@ On the other hand, if you're looking at JavaScript documentation that tells you
 
 If you want to set a property, you can use the `prop` element modifier.
 
-## Calling Methods On First Render
+## Calling Methods On Render
 
-So far, we've talked about web APIs that work by setting attributes as well as web APIs that work by setting properties. But what about web APIs that work by calling methods, like setting focus on an element?
+So far, we've talked about web APIs that work by setting attributes as well as web APIs that work by setting properties. But what about web APIs that work by calling methods, like setting focus on an element? 
 
 For example, let's say we want to focus an `<input>` in a form as soon as the form is rendered. The web API for focusing an element is:
 
 ```js
 inputElement.focus();
 ```
 
-This code needs to run after the element is rendered.
+We can use a modifier to run a method immediately after an element has rendered.
+To follow along with the examples below, make sure you have `ember-modifier` installed in your app:
 
-The simplest way to accomplish this is by using the `did-insert` modifier from [@ember/render-modifiers][render-modifiers].
-
-[render-modifiers]: https://github.com/emberjs/ember-render-modifiers
-
-```handlebars {app/components/edit-form.hbs}
-<form>
-  <input {{did-insert this.focus}}>
-</form>
+```
+npx ember install ember-modifier
```

Comment:
whoops, this should be ember-cli

this package: https://www.npmjs.com/package/ember
only prints this message: https://github.com/emberjs/ember/blob/master/index.js

and doesn't actually do anything.
Would be great if `ember` were configured as a bin that delegated to `npx ember-cli`

Reply:
Oh right, thank you. I will change it to use global ember for now.